.github/workflows/feature-branch.yml

```diff
@@ -0,0 +1,99 @@
+name: Feature branch CI
```

Comment:
It's not good when the name relates strictly with action. `Feature branch` will last much much longer. Even if we decide to change triggers to `pull_request` instead of `push`.

Reply:
Yes, but now it's that name in all repositories.
Therefore I would keep the naming and eventually change it in all repos at once if this really will be needed.

Reply:
Ok